bin/build_config

```diff
@@ -9,9 +9,10 @@ require 'rubocop-rspec'
 require 'rubocop/rspec/description_extractor'
 require 'rubocop/rspec/config_formatter'
 
-glob = File.join(__dir__, '..', 'lib', 'rubocop', 'cop', 'rspec',
+glob = File.join('lib', 'rubocop', 'cop', 'rspec',
                  '{,capybara,factory_bot,rails}', '*.rb')
-YARD.parse(Dir[glob], [])
+rspec_cop_path = File.join('lib', 'rubocop', 'cop', 'rspec', 'cop.rb')
+YARD.parse([rspec_cop_path, *Dir[glob]], [])
```

Comment:
Since `'{,capybara,factory_bot,rails}', '*.rb')` already covers `cop.rb`, perhaps it should be made more explicit that we _want_ the `cop.rb` entry to be first. Maybe by using `[]#prepend`, or by adding a comment?